src/containers/MoveListView.js

```diff
@@ -184,76 +206,82 @@ class MoveListView extends React.Component {
 
 	render() {
 		return (
```

Comment:
Don't look too much on the changes below here. Spacing got changed accidentally but I didn't change anything other than adding the prop onDragEnd to all the MoveList components.